inc/namespace.php

```diff
@@ -10,9 +10,12 @@
 function bootstrap() {
 	register_cache_groups();
 
-	if ( ! is_installed() ) {
-		create_tables();
+	if ( ! is_installed() && ! create_tables() ) {
+		return;
```

Comment:
maybe run bootstrap `wp_install` action.

Reply:
Tried this but `wp_install` fires before network install is complete so the option would end up in the `wp_options` table rather than the `wp_sitemeta` table where it belongs.

After a quick search I couldn't find an appropriate hook to use as an alternative so left it.

However, I did end up with a jobs table that looked like this after install which is not ideal.

<img width="413" alt="_MySQL_5_7_25-0ubuntu0_16_04_2__Chassis_wordpress_wp_cavalcade_jobs" src="https://user-images.githubusercontent.com/519727/56453236-bd0aa300-6382-11e9-9206-171f2e72e6ec.png">